shared/util/src/protocol/agent/agent.protocol.providers.ts

```diff
@@ -688,9 +697,120 @@ export interface FetchThirdPartyPullRequestPullRequest {
 	};
 	participants: {
 		nodes: {
-			avatarUrl: string;
+			//gitlab
+			avatarUrl?: string;
+			//bitbucket
+			type?: string;
+			user: {
+				display_name: string;
+				links: {
+					avatar: {
+						href: string;
+					};
+				};
+				type?: string;
+				uuid?: string;
+				account_id: string;
+				nickname: string;
+			};
+			role: string;
+			approved: boolean;
+			state: null | string;
```

Comment:
Unless there is a really compelling reason, change all the `null | string` in these new types to be more use the typescript optional notation. It requires changing a few lines of code in `butbucket.ts` but it is not too bad

```suggestion
			state?: string;
```

Reply:
the state options that come back from bitbucket are "approved", "changes_requested" or null.  That's why I have it as null | string, because null is a legitimate option there.